iot_config/src/route.rs

```diff
@@ -241,43 +247,49 @@ pub async fn update_euis(
 ) -> Result<(), RouteStorageError> {
     let mut transaction = db.begin().await?;
 
-    if !to_add.is_empty() {
-        insert_euis(to_add, &mut transaction).await?;
-    }
+    let added_euis = if !to_add.is_empty() {
+        insert_euis(to_add, &mut transaction).await?
+    } else {
+        vec![]
+    };
```

Comment:
creating an empty vec is pretty low cost, so this isn't inherently bad, but it does stick out.